scss/tokens/_raw.scss

```diff
@@ -1,82 +1,407 @@
+$ouds-font-line-height-950: 44px !default;
+$ouds-font-line-height-1050: 48px !default;
+// $ouds-font-line-height-1150: 52px !default;
+$ouds-font-line-height-1250: 56px !default;
+// $ouds-font-line-height-1350: 60px !default;
+$ouds-font-line-height-1450: 64px !default;
+$ouds-font-line-height-1850: 72px !default;
+$ouds-font-line-height-2050: 80px !default;
+$ouds-font-size-150: 12px !default;
+// $ouds-font-size-175: 13px !default;
+$ouds-font-size-200: 14px !default;
+$ouds-font-size-250: 16px !default;
+$ouds-font-size-300: 18px !default;
+$ouds-font-size-350: 20px !default;
+$ouds-font-size-450: 24px !default;
+$ouds-font-size-550: 28px !default;
+$ouds-font-size-650: 32px !default;
+$ouds-font-size-750: 36px !default;
+$ouds-font-size-850: 40px !default;
+// $ouds-font-size-950: 44px !default;
+$ouds-font-size-1050: 48px !default;
+// $ouds-font-size-1150: 52px !default;
+$ouds-font-size-1250: 56px !default;
+$ouds-font-size-1450: 64px !default;
+$ouds-font-size-1850: 72px !default;
+// $ouds-font-weight-100: 100 !default;
+// $ouds-font-weight-200: 200 !default;
+// $ouds-font-weight-300: 300 !default;
+$ouds-font-weight-400: 400 !default;
+// $ouds-font-weight-500: 500 !default;
+// $ouds-font-weight-600: 600 !default;
+$ouds-font-weight-700: 700 !default;
+// $ouds-font-weight-800: 800 !default;
+// $ouds-font-weight-900: 900 !default;
+// $ouds-font-weight-950: 950 !default;
+// scss-docs-end ouds-raw-font
 
-$ouds-elevation-y-0:    0 !default;
-$ouds-elevation-y-100:  1px !default;
-$ouds-elevation-y-200:  2px !default;
-$ouds-elevation-y-300:  4px !default;
-// $ouds-elevation-y-400:  8px !default;
-$ouds-elevation-y-500:  12px !default;
-// $ouds-elevation-y-600:  20px !default;
-// scss-docs-end ouds-raw-elevation
+// Grid
+
+// scss-docs-start ouds-raw-grid
+// $ouds-grid-column-count-100: 1px !default;
+// $ouds-grid-column-count-200: 2px !default;
+$ouds-grid-column-count-400: 4px !default;
+$ouds-grid-column-count-600: 6px !default;
+// $ouds-grid-column-count-800: 8px !default;
+// $ouds-grid-column-count-1000: 10px !default;
+$ouds-grid-column-count-1200: 12px !default;
+$ouds-grid-max-width-100: 389px !default;
+$ouds-grid-max-width-200: 479px !default;
+$ouds-grid-max-width-300: 735px !default;
+$ouds-grid-max-width-400: 1023px !default;
+$ouds-grid-max-width-500: 1319px !default;
+$ouds-grid-max-width-600: 1639px !default;
+$ouds-grid-max-width-650: 1680px !default;
+$ouds-grid-max-width-700: 1879px !default;
+$ouds-grid-max-width-800: 1920px !default;
+$ouds-grid-min-width-100: 1px !default;
+$ouds-grid-min-width-200: 390px !default;
+$ouds-grid-min-width-300: 480px !default;
+$ouds-grid-min-width-400: 736px !default;
+$ouds-grid-min-width-500: 1024px !default;
+$ouds-grid-min-width-600: 1320px !default;
+$ouds-grid-min-width-700: 1640px !default;
+$ouds-grid-min-width-800: 1880px !default;
+$ouds-grid-width-100: 360px !default;
+$ouds-grid-width-200: 390px !default;
+$ouds-grid-width-300: 480px !default;
```

Comment:
Should these variables be generated using `$ouds-dimension-base`?

Reply:
We need to double-check with the designers, but it wasn't mentioned like that in the specifications.